app/jobs/email_csv_job.rb

```diff
@@ -30,7 +30,8 @@ def perform(user, search_term = nil, filters = {}, all_orgs = false, organisatio
     storage_service.write_file(filename, BYTE_ORDER_MARK + csv_string)
     csv_download = CsvDownload.create!(user:, organisation: user.organisation, filename:, download_type: log_type, expiration_time: EXPIRATION_TIME)
 
-    url = download_csv_download_url(csv_download.id, host: ENV["APP_HOST"])
+    binding.pry
+    url = csv_download_url(csv_download.id, host: ENV["APP_HOST"])
 
```

Comment:
Still can't see the page but can download. There's a binding.pry here

Reply:
Ah, sorry about that... Updated it!